src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -130,16 +139,30 @@ abstract class AbstractNormalizer implements NormalizerInterface, DenormalizerIn
 
     /**
      * @var NameConverterInterface|null
+     *
+     * @deprecated since Symfony 5.1, use $nameConverters instead.
      */
     protected $nameConverter;
 
+    /**
+     * @var array<string, NameConverterInterface>
+     */
+    protected $nameConverters = [];
+
     /**
      * Sets the {@link ClassMetadataFactoryInterface} to use.
      */
-    public function __construct(ClassMetadataFactoryInterface $classMetadataFactory = null, NameConverterInterface $nameConverter = null, array $defaultContext = [])
+    public function __construct(ClassMetadataFactoryInterface $classMetadataFactory = null, /* array */ $nameConverter/*s*/ = [], array $defaultContext = [])
```

Comment:
I'm not sure if it's possible to change or deprecate the name of a constructor argument since it could be used by DI with a named argument.